view/frontend/web/js/view/catalog/badge.js

```diff
@@ -80,6 +92,30 @@ define([
         return price ;
     }
 
+    /**
+     *
+     * @param productId Id of product for Id concatenation
+     * @param subClass Sub class to select
+     * @returns {*}
+     */
+    function getHtmlPriceContainer(productId,subClass = false){
+        var classToSelect = ''
```

Comment:
I know `;` are not required ... but can you add it anyway please

Reply:
Done